Plugins/Flow.Launcher.Plugin.Program/Languages/en.xaml

```diff
@@ -61,7 +61,7 @@
         Insert file suffixes you want to index. Suffixes should be separated by ';'. (ex>bat;py)
     </system:String>
     <system:String x:Key="flowlauncher_plugin_program_protocol_tooltip">
-        Insert protocols of .url files you want to index. Protocols should be separated by ';'. (ex>ftp;netflix)
+        Insert protocols of .url files you want to index. Protocols should be separated by ';', and should end with "//". (ex>ftp://;mailto://)
```

Comment:
Don't want really parse the URL. So try to tell users how to avoid situaions like `http` actually matches a URL starts with `https`.

Reply:
why not parse it? Uri.TryParse is not expensive?

Reply:
Because for steam games their URL scheme is `steam://app/` rather than `steam://` so parsing the URL for protocol is not enough.